_changelogs/1.1.0.md

```diff
@@ -0,0 +1,27 @@
+## Changes
+
+* __BREAKING__ : Environment variables became new field `skip_if_empty`, if this property is set and the value is empty, the Environment will not exported during `envman run`. *This is the same behavior as previous version of envman worked*. To add env with skip_if_empty, just call `envman add --key KEY --value VALUE --skip-if-empty`.
```

Comment:
Environment variables got a new option: `skip_if_empty`. If this property is set/enabled and the value of the environment variable is empty, the Environment will not be exported during `envman run`. **This is the same behavior as previous version of envman worked**. If this option is not enabled (**the new default behaviour**) then the empty value will be used as the value, it won't be skipped. Practically this means that if you add two environment variables to envman with the same key and the second one has an empty value : if you enable this option (`skip_if_empty`) the first value will be used (the second, empty one will be skipped), and if you disable this option (the new default) the empty value will be used, it'll overwrite the first non-empty value. To add env with skip_if_empty, just call `envman add --key KEY --value VALUE --skip-if-empty`.